### DISCUSSION TITLE
Prisma faster?

### DISCUSSION THREAD

**Discussion by rajamoulimallareddy** [prisma/prisma#16651] [Q&A]
Is there any benchmark for prisma when compared with mongodb, SQL directly? And is it faster or slower?

**Comment by nurul3101** [comment]
Hi @rajamoulimallareddy 👋 

Here is a community-created comparison [blog](https://dev.to/josethz00/benchmark-prisma-vs-typeorm-3873) which you can refer to. 
You might also want to have a look at this [Comparison of ORMs in Node Ecosystem](https://www.prisma.io/dataguide/database-tools/top-nodejs-orms-query-builders-and-database-libraries).

**Reply by vladi-stevanovic** [reply]
👋 Hi @rajamoulimallareddy , did you perhaps have a chance to check Nurul's recommendation? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
